acl-lib/src/cmr/acl/core.clj

```diff
@@ -159,21 +159,31 @@
            error-message))
         nil))))
 
+(defn- valid-token?
+  "test"
+  [context]
+  (if-let [token (:token context)]
+    (try
+      (= 200 (first (echo-tokens/get-token-info context token)))
+      (catch Exception e false))
```

Comment:
If the return code is not 200 it probably makes sense for us to log the error message that comes back.